### ISSUE TITLE
Webview with content scripts crashes on load non .swf object (v0.24.0, Win7, x64)

### ISSUE THREAD

**Issue by yss50829** [nwjs/nw.js#6075]
**Required conditions:**
1. nwjs v0.24.0 (works on 0.23.7)
2. webview with .addContentScripts( { ... , run_at: 'document_start', 'all_frames' : true })
3. page with `<object data="{url_without_swf_extension}"></object>` 

**Not important:**
1. sdk/non-sdk
2. x64/ia32
3. flash enabled/disabled
4. response for object url

**Steps for reproducing:**
1. launch nw.exe (sdk-v0.24.0)
2. open devtools (F12)
3. run in console following code
```
( swf => {
    const webview = document.createElement('webview');        
    
    webview.addContentScripts([{
        name: 'test',
        matches: ['<all_urls>'],
        js: { code: 'console.log(location.href)' },
	all_frames: true, /* important */
        run_at: 'document_start' /* important */
    }])
    webview.src = 'http://example.org/';
    webview.style.cssText = 'width: 100%; height: 100%; display: block;';
    webview.addEventListener('contentload', () => {
        webview.executeScript({ code: `document.body.innerHTML = '<object data="/test${swf ? '.swf' : ''}"></object>';`})
    } );
    webview.addEventListener( 'exit', e => console.log( 'exit', e ) );

    document.body.innerHTML = '';
    document.body.appendChild( webview );
})( false /* add swf extension to object url or not */ ); 
```

**Comment by Christywl** [comment]
I can reproduce this issue on Linux/Windows with nwjs-sdk-v0.24.0. 
![image](https://user-images.githubusercontent.com/20039821/28949862-3697c090-78f2-11e7-9884-ba37f0895f45.png)

**Comment by yss50829** [comment]
It seems like fixed in 0.25.0. Should it be closed?

**Comment by Christywl** [comment]
Thanks for the update. It also works fine on my Linux/Windows with nwjs-sdk-v0.25.0. I think we can close it now. If meet again later, you could reopen it or report a new one.
